llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1696,3 +1702,79 @@ void VPlanTransforms::createInterleaveGroups(
       }
   }
 }
+
+void VPlanTransforms::convertToMultiCond(VPlan &Plan, ScalarEvolution &SE,
```

Comment:
It's not immediately obvious from the name what this means and is used for. I wonder if it's better named as something like `handleUncountableEarlyExit` to reflect what it will actually be used for? Or perhaps `handleMultipleRegionSuccessors`?